dfx/index.ts

```diff
@@ -75,11 +105,16 @@ export function getIdentities(): string[] {
     return identities;
 }
 
+export function identityExists(name: string): boolean {
+    const identities = getIdentities();
+    return identities.includes(name);
+}
+
 export function removeIdentity(name: string) {
     execSync(`dfx identity remove ${name}`);
 }
 
-export async function getIdentity(
+export async function getIdentityFromPemFile(
```

Comment:
This should probably rename with secp256k1 in mind, or it should accept as a parameter the type of key, since this won't work generally

Reply:
This one isn't used anymore and is essentially the same but less powerful as the other getIdentity function. I just got rid of it instead.